packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx

```diff
@@ -64,18 +68,24 @@ const LikeButton = ({
   />
 );
 
-const ForkButton = ({ signals, secondary, style }: ButtonProps) => (
+const ForkButton = ({
+  signals,
+  secondary,
+  isForking,
+  style,
+}: ForkButtonProps) => (
   <Button
     onClick={() => {
       signals.editor.forkSandboxClicked();
     }}
     style={style}
     secondary={secondary}
+    disabled={isForking}
     small
   >
     <>
       <Fork style={{ marginRight: '.5rem' }} />
-      Fork
+      {isForking ? 'Forking' : 'Fork'}
```

Comment:
I think it would be best to name it "Forking..." with the three dots to indicate an action.

Reply:
Yes, i was hesitating between both because the dots makes the button really larger while forking. Now you convinced me :)